.github/workflows/borg_integration_tests.yml

```
@@ -0,0 +1,185 @@
+          - os: macos-14
+            test-type: os-matrix
+            borg-version: 1.4.1
+
+    name: ${{ matrix.test-type }} - ${{ matrix.os }} - Borg ${{ matrix.borg-version }}
+    
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Setup Go
+        uses: actions/setup-go@v4
```

actionlint (1.7.7)

the runner of "actions/setup-go@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)